reproducibility_project/src/engines/cassandra/project.py

```diff
@@ -376,6 +376,72 @@ def run_cassandra(job):
         )
 
 
+@Project.operation
+@Project.pre(cassandra_complete)
+@Project.post(lambda job: "mean_energy_box1" in job.document)
+def statistics(job):
+    """Compute statistical quantities for each job."""
+    import ele
+    from unyt import angstrom, g, mole
+
+    proplist = [
+        "energy_total",
+        "volume",
+        "nmols",
+        "pressure",
+        "density",
+        "mass_density",
+        "energy_angle",
+        "energy_dihedral",
+        "energy_intravdw",
+        "energy_intraq",
+        "energy_inter",
+        "energy_intervdw",
+        "energy_lrc",
+        "energy_interq",
+        "energy_recip",
+        "energy_self",
+        "enthalpy",
+    ]
+
+    NA = 6.022e23
+
+    C = ele.element_from_symbol("C")
+    H = ele.element_from_symbol("H")
+
+    mw = C.mass + H.mass * 4
+    mw *= g / mole
```

Comment:
This part isn't needed anymore, and would probably just serve to confuse other readers.
```suggestion

```